superset/security/manager.py

```diff
@@ -726,7 +726,8 @@ def create_custom_permissions(self) -> None:
         self.add_permission_view_menu("can_csv", "Superset")
         self.add_permission_view_menu("can_share_dashboard", "Superset")
         self.add_permission_view_menu("can_share_chart", "Superset")
-        self.add_permission_view_menu("can_view_and_drill", "Dashboard")
```

Comment:
I know that `can_view_and_drill` permission wasn't around for long so probably few people have used it, but shouldn't we create a migration for it?

Reply:
It'd be nice to have a reusable function for perm migration that we could use in database migrations. Now that we have `migration_utils.py` in theory it could contain a function `migrate_permission(before_perm, after_perm)`. Though in theory it should only be use for equivalent (rename) or more atomic/selective permissions.

Reply:
we have this: https://github.com/apache/superset/blob/master/superset/migrations/shared/security_converge.py#L215

Reply:
hi I have all roles. And can not see drill down menu in superset graphs. Why?